packages/console/src/pages/Cloud/components/ProgressBar/index.module.scss

```diff
@@ -0,0 +1,16 @@
+@use '@/scss/underscore' as _;
+
+.progressBar {
+  display: flex;
+  gap: _.unit(1);
+
+  .stepIndicator {
+    height: 4px;
+    background-color: var(--color-neutral-variant-80);
+    flex: 1;
+
+    &.active {
+      background-color: var(--color-text-link);
```

Comment:
It would be better to add some animation here, like having the purple color growing from 0 to 100% in a second

Reply:
The representations of these interactions are complicated, and we cannot use this simple amination, for it is on different pages and may navigate back and force.